dist/index.html

```diff
@@ -13,11 +13,16 @@ <h1 class="main_title">Leaderboard app</h1>
 
       <div class="score-leaderboard">
```

Comment:
Please add project descriptions and instructions to run the project locally for a non-technical user in the README file

Reply:
Thanks! Working on this!